Fdc3 for web impl test policy (#1480)
finos/FDC3: CONTRIBUTING.md

To contribute a patch for a New Intent, see the [Submit New Intent](docs/guides/submit-new-intent) guide.

**3.6. Test Policy**

- The FDC3 project takes very seriously code quality, especially with respect to production-grade sub-modules.  This includes:

|Module | Test Policy |
|---|---|
|`packages/fdc3-agent-proxy` | Contributions must improve or maintain coverage at the current levels |
|`packages/fdc3-get-agent` |Contributions must improve or maintain coverage at the current levels 
|`packages/fdc3-commonjs` | Code should not be added here |
|`packages/fdc3-context` | Must contain only JSON Schemas |
|`packages/fdc3-schema` | Must contain only JSON Schemas |
|`packages/fdc3-context` | Must contain only JSON Schemas |
|`packages/fdc3-standard` | Contributions must improve or maintain coverage at the current levels |
|`toolbox/fdc3-for-web/fdc3-web-impl` | Contributions must improve or maintain coverage at the current levels |

Please note that the FDC3 project uses the following test policy:

>   * All code must be tested (coverage is reported in the PR comments)

Author: yes... I didn't mean to say that every line should be covered.  Is this how it came across?

>   * All code must be tested (coverage is reported in the PR comments)

Reviewer: Yeah, to me 'all code must be tested' sounds like every line and branch (which is tough). Perhaps a qualifier like 'All code _units_ or _modules_ ...'.

Perhaps some slight softening would also make sense (replace 'must' with 'should')?

>   * All code must be tested (coverage is reported in the PR comments)

Author: ```suggestion
  * Code should have tests (coverage is reported in the PR comments)
```

  * All tests must pass. (checked automatically by the PR comments)

>   * All new code must be covered by tests.

Author: ```suggestion
```
